### ISSUE TITLE
[V3 Core] Unable to load streams cog with a set mixer alerts

### ISSUE THREAD

**Issue by ASSASSIN0831** [Cog-Creators/Red-DiscordBot#1932]
# Command bugs

#### Command name

[p]load streams

#### What cog is this command from?

Core and Streams

#### What were you expecting to happen?

For the streams cog to load.

#### What actually happened?
I added a mixer stream and it worked but I updated to version 17 and noticed the streams cog wasn't loaded. so I did the load command and got this error. I deleted the json file and then it loaded jsut fine. Once I added a mixer stream the tried to reload the cog by first doing the unload command then doing the load command. I got the error again.

```
Exception during loading of cog
Traceback (most recent call last):
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/core/core_commands.py", line 87, in _load
    await bot.load_extension(spec)
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/core/bot.py", line 222, in load_extension
    await lib.setup(self)
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/cogs/streams/__init__.py", line 6, in setup
    await cog.initialize()
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/cogs/streams/streams.py", line 71, in initialize
    self.streams = await self.load_streams()
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/cogs/streams/streams.py", line 611, in load_streams
    token = await self.db.tokens.get_raw(_class.__name__)
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/core/config.py", line 342, in get_raw
    return await self.driver.get(*self.identifiers, *path)
  File "/home/jjay/rewrite/lib/python3.6/site-packages/redbot/core/drivers/red_json.py", line 100, in get
    partial = partial[i]
KeyError: 'MixerStream'
```

#### How can we reproduce this issue?

1. Add a mixer stream alert
2. Unload the stream cog
3. Load the streams cog

**Comment by Tobotimus** [comment]
Hi, thanks for reporting this issue! I've set the milestone for the next beta (19), and I'll assign myself to try and get it done by then :)

**Comment by Tobotimus** [comment]
Resolved by #1994.
